libnmstate/ifaces/ifaces.py

```diff
@@ -140,6 +141,25 @@ def _bring_slave_up_if_not_in_desire(self):
                         slave_iface.mark_as_up()
                         slave_iface.mark_as_changed()
 
+    def _validate_ovs_patch_peers(self):
+        """
+        When OVS patch peer does not exist or is down, raise an error.
+        """
+        for iface in self._ifaces.values():
+            if iface.is_patch_port:
+                for peer_name in iface.peers:
```

Comment:
Both the schema and NM doc define it as string, not list.

Reply:
Done, thanks!